src/Uno.UI/ts/Windows/Storage/Pickers/FileSavePicker.ts

```diff
@@ -14,8 +14,14 @@
 			const options: SaveFilePickerOptions = {
 				excludeAcceptAllOption: !showAllEntry,
 				types: [],
+				id: id,
+				startIn: startIn
```

Comment:
What happens if the current browser does not support it ? Is it ignored ?

Reply:
Yes, unknown properties seem to be properly ignored and have no effect.